### PR TITLE
refactor(smali): refactor queries from upstream

### PR DESCRIPTION
**Pull request by amaanq**
:musical_note:

### CODE REVIEW

File: queries/smali/highlights.scm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 (primitive_type) @type.builtin
 
 ((class_identifier) @type.builtin
-  (#vim-match? @type.builtin "^L(android|com/android|dalvik|java)/"))
+  (#match? @type.builtin "^L(android|com/android|dalvik|java|kotlinx)/"))
```
Comment (lucario387) [inline queries/smali/highlights.scm:8]: can't this be any-of?

Comment (amaanq) [inline queries/smali/highlights.scm:8]: no, ".super Ljava/lang/Object;"

File: queries/smali/highlights.scm [hunk 2]
```diff
@@ -183,9 +177,6 @@
 
 (comment) @comment @spell
 
-(class_definition
-  (comment) @comment.documentation)
-
```
Comment (lucario387) [inline queries/smali/highlights.scm:188]: So..., what is the alternative, or was it unnecessary from the beginning?

Comment (amaanq) [reply queries/smali/highlights.scm:188]: Should not have been removed, oops

**Review by lucario387 — APPROVED** [review]
Seeing as you are the maintainer of smali too, I don't see why I shouldn't trust your changes :)

**Pull request merged**